xbout/tests/test_against_collect.py

```diff
@@ -2,45 +2,107 @@
 
 import numpy.testing as npt
 
-from xbout.load import _auto_open_mfboutdataset
+from xbout.load import open_boutdataset
+from .test_load import create_bout_ds, create_bout_ds_list, METADATA_VARS
 
+from boutdata import collect
```

Comment:
Does this mean anyone who wants to run the test suite has to have `boutdata.collect` on their machine? If so I think that's okay.

Reply:
I guess it does. It would probably be good to make it optional - the travis tests already pip-install boutdata, so will be checking this test even if users don't.